src/Microsoft.DotNet.Wpf/src/PresentationUI/MS/Internal/Documents/SignatureResourceHelper.cs

```diff
@@ -89,12 +89,7 @@ internal static class SignatureResourceHelper
                 sigStatus = SignatureStatus.Invalid;
             }
 
-            string resourceName = string.Format(
-                CultureInfo.InvariantCulture,
-                @"{0}_{1}x{2}",
-                sigStatus.ToString(),
-                height,
-                width);
+            string resourceName = $@"{sigStatus}_{height}x{width}";
```

Comment:
Missing InvariantCulture here.

Reply:
In the winforms repo, when doing such refactorings, a choice was made to ignore the invariant culture for int. I am no specialist but apparently it has no impact on integer formatting. I can put it back just for peace of mind

Reply:
Correct, as long as there is no format requested, the culture is not used (other than for negative sign, which shouldn't be an issue for width/height). 

https://github.com/dotnet/runtime/blob/2f43856e78faebe7347fdd76acc26de7ffaec539/src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs#L919-L925